### ISSUE TITLE
Enable upload of requirements.yml

### ISSUE THREAD

**Issue by chouseknecht** [ansible/galaxy_ng#258]
pulp_ansible supports an optional `requirements.yml` file path that essentially enables the user to allowlist content to be mirrored from the remote. It assumes, however, that the file path is accessible on the filesystem local to the backend processes, which might be difficult, if not impossible, for the end user to access.

GalaxyNG should enhance pulp_ansible by making it possible for the end user to upload a `requirements.yml` file through the UI.

**Comment by chouseknecht** [comment]
Subtask of #22

**Comment by alikins** [comment]
https://pulp-ansible.readthedocs.io/en/latest/restapi.html#operation/pulp_api_v3_remotes_ansible_collection_list  
(POST http://localhost:24817/pulp/api/v3/remotes/ansible/collection/)  takes a 'requirements_file' attribute that I believe the value of can be the contents of a requirements.yaml file.

**Comment by chouseknecht** [comment]
The UI work is already covered in #281

**Comment by rochacbruno** [comment]
Currently `requirements_file` has a limitation of 255 chars

https://github.com/pulp/pulp_ansible/blob/b3695faf48c94dd633baf8e775e5926e09dd85fc/pulp_ansible/app/models.py#L218

I will send a PR removing that limitation to unblock this issue.

**Comment by rochacbruno** [comment]
@chouseknecht can we close/update this one? this is already implemented in /remotes API
